tests/wpwebdriveracceptance/DatePicker3Cept.php

```diff
@@ -0,0 +1,175 @@
+
+<?php
+
+$I = new WpwebdriveracceptanceTester($scenario);
+$I->wantTo('Test the datepicker cases');
+
+//Login to backend
+$I->amOnPage("/wp-login.php");
+$I->fillField('Username', 'test_user');
+$I->fillField('Password','test_user');
+//$I->fillField("#user_login", "admin");
+//$I->fillField("#user_pass", "admin");
+$I->click('Log In');
+
+$I->wait(5);
+
+$output = '';
+
+$result = '';
+
+$array = array("2016-01-15","1/15/2016","01/15/2016","15/1/2016","15/01/2016","1-15-2016","01-15-2016","15-1-2016","15-01-2016");
```

Comment:
Whitespace! Be nice to add space after opening and before closing parentheses, and after commas :smile: